chrome_extension/mock_client.py

```diff
@@ -0,0 +1,51 @@
+"""Mock client for testing the server."""
```

Comment:
What do you think about adding a line about why this is useful, and another one with example usage?